tests/test_app/templates/app.html

```diff
@@ -1 +1 @@
-{{ page|json_script:"page" }}
+{{ page|json_script:"__page" }}
```

Comment:
Here you missed:

```suggestion
{{ __page|json_script:"__page" }}
```

But I have discovered that Django prevent using variables starting with `__`:
```
Variables and attributes may not begin with underscores: '__page'
```

I propose to use `page__` instead of using `__page`.